app-admin/stown/stown-1.2.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+# shellcheck shell=bash disable=2034
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=(python3_{10..13})
+
+inherit distutils-r1 readme.gentoo-r1
+distutils_enable_tests unittest
+
+DESCRIPTION="Manage file system object mapping via symlinks"
+HOMEPAGE="https://www.seichter.de/stown/"
+SRC_URI="https://github.com/rseichter/stown/archive/refs/tags/${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DOCS=("${S}"/README.md)
```

Comment:
Gentoo style is usually spaces in arrays, e.g. `DOCS=( "${S}"/README.md )`, but are you sure this is needed? The default should work fine.

Reply:
Fixed (removed)